nextcloud-aio-helm-chart/templates/nextcloud-aio-networkpolicy.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if eq .Values.NETWORK_POLICY_ENABLED "yes" }}
+# https://github.com/ahmetb/kubernetes-network-policy-recipes/blob/master/04-deny-traffic-from-other-namespaces.md
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  namespace: "{{ .Values.NAMESPACE }}"
+  name: nextcloud-aio-deny-from-other-namespaces
+spec:
+  podSelector:
+    matchLabels:
+  ingress:
+  - from:
+    - podSelector: {}
+{{- end }}
```

Comment:
```suggestion
{{- if eq .Values.NETWORK_POLICY_ENABLED "yes" }}
# https://github.com/ahmetb/kubernetes-network-policy-recipes/blob/master/04-deny-traffic-from-other-namespaces.md
kind: NetworkPolicy
apiVersion: networking.k8s.io/v1
metadata:
  namespace: "{{ .Values.NAMESPACE }}"
  name: nextcloud-aio-deny-from-other-namespaces
spec:
  podSelector:
    matchLabels:
  policyTypes:
    - Ingress
    - Egress
  ingress:
  - from:
    - podSelector: {}
  egress:
  - to:
    - podSelector: {}
{{- end }}
```

We should also allow any egress inside the namespace. I tested the adjusted policy locally. Health checks will fail otherwise if egress is denied by default on the cluster.

I added the `egress` block below the `ingress` one and added the `policyTypes` array. The policy wasn't modified otherwise.

Reply:
Thanks Richard :)

Done with https://github.com/nextcloud/all-in-one/pull/5024/commits/533c270d061e45f867654600bdd5e47f76e76b10